frontend/packages/jobs/src/functions/processGenerateReview.ts

```diff
@@ -1,12 +1,14 @@
 import { prisma } from '@liam-hq/db'
 import { getFileContent } from '@liam-hq/github'
+import type { InferOutput } from 'valibot'
 import { generateReview } from '../prompts/generateReview/generateReview'
+import type { reviewSchema } from '../prompts/generateReview/reviewSchema'
 import type { GenerateReviewPayload } from '../types'
 import { langfuseLangchainHandler } from './langfuseLangchainHandler'
 
 export const processGenerateReview = async (
   payload: GenerateReviewPayload,
-): Promise<string> => {
+): Promise<InferOutput<typeof reviewSchema>> => {
```

Comment:
FYI: This type seems to be used a lot, so it is better to add a type definition.

```ts
export type Review = InferOutput<typeof reviewSchema>
```